### PR TITLE
Minh development

### PR DESCRIPTION
**Pull request by minhhuynguyen91**

### CODE REVIEW

File: app/models/match.rb [hunk 1]
```diff
@@ -1,3 +1,17 @@
 class Match < ApplicationRecord
   belongs_to :field
+  
+  def self.update_match_data(req_id)
+    m_request = MatchRequest.find_by_id(req_id)
+    m = Match.find_by_id(m_request.match_id)
+    if m_request.status == "ACCEPT"
+      m.team_away_id = m_request.team_request_id
```
Comment (minhhuynguyen91) [inline app/models/match.rb:8]: Please help me to change this line @lanhhoang 
`m.team_away_id = m_request.team_request_id` to `m.team_away_id = m_request.requested_team_id`

Comment (minhhuynguyen91) [inline app/models/match.rb]: @lan

**Comment by minhhuynguyen91** [comment]
[Match](https://trello.com/c/KbY0G73v/16-match)

**Comment by lanhhoang** [comment]
Copy code to #22

**Pull request closed without merging**